src/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ set(LIBRARY_PUBLIC_HEADERS
 add_library(${LIBRARY_NAME} SHARED ${LIBRARY_SOURCES})
 add_library(${PROJECT_NAME}::${LIBRARY_NAME} ALIAS ${LIBRARY_NAME})
 target_include_directories(${LIBRARY_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/cwrap")
-target_link_libraries(${LIBRARY_NAME} ${OPENSSL_LIBRARIES})
```

Comment:
You might want to bump the required CMake version to 3.5. (https://gitlab.kitware.com/cmake/cmake/commit/3f9b081f6ee85e0691c36496d989edbe8382589d)

Reply:
Done